src/SciAssist/utils/windows_pdf2text.py

```diff
@@ -19,7 +19,8 @@ def process_pdf(path):
     for page_layout in extract_pages(path):
         for element in page_layout:
             if "get_text" in dir(element):
-                text = element.get_text().replace("\n","")
+                text = element.get_text().replace("-\n","")
```

Comment:
remove hyphen in the end of one line.

Reply:
Yes the problem only appears for windows, not for linux.